<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TocSideBar</class>
 <widget class="QWidget" name="TocSideBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLineEdit" name="findEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="fNextButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/kiwix.qrc">
         <normaloff>:/icons/search_forward.svg</normaloff>:/icons/search_forward.svg</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fPreviousButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/kiwix.qrc">
         <normaloff>:/icons/search_backward.svg</normaloff>:/icons/search_backward.svg</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/kiwix.qrc"/>
 </resources>
 <connections/>
</ui>
